xml/System.Threading/Interlocked.xml

```diff
@@ -1097,6 +1098,7 @@ If `comparand` and the object in `location1` are equal by reference, then `value
         <exception cref="T:System.ArgumentNullException">The address of <paramref name="location" /> is a <see langword="null" /> pointer.</exception>
         <related type="Article" href="/dotnet/standard/threading/">Managed Threading</related>
         <related type="Article" href="/dotnet/standard/threading/overview-of-synchronization-primitives">Overview of synchronization primitives</related>
+        <exception cref="T:System.NullReferenceException">The address of location is a <see langword="null" /> pointer.</exception>
```

Comment:
This and a few others don't have param ref. Assuming this is because the source doesn't have them.

```suggestion
        <exception cref="T:System.NullReferenceException">The address of <paramref name="location" /> is a <see langword="null" /> pointer.</exception>
```